README.md

```diff
@@ -279,7 +279,38 @@ To use the WAF service, use the API Token provided to you.
 An access rule identifies legitimate traffic and threats by ASN, Cookie, Country, 
 IP Address, Referrer, URL, User agent, HTTP method, Media type, File extension, 
 and Request headers.
-```
+
+For detailed information about Access Rules in WAF, please read the [official documentation](https://docs.edgecast.com/cdn/#Web-Security/Access-Rules.htm).
+
+#### Bot Rules
+Text TBD
```

Comment:
We just need a ticket (or tie this to the SDK ticket) to update once we have the documentation from Jose.

Reply:
Updated